src/Utility/Utilities.C

```diff
@@ -445,6 +445,16 @@ std::set<int> utl::getDigits (int num)
 }
 
 
+int utl::getDirs (int ncmp)
+{
+  int res = 0, mul = std::pow(10,ncmp-1);
+  for (int i = 1; i <= ncmp; ++i, mul /= 10)
+    res += i*mul;
```

Comment:
Sub-sub optimizing, but this would result in the same without the pow-function(?)
```
  int res = 0, mul = 1;
  for (int i = ncmp; i >= 1; i--, mul *= 10)
    res *= i*mul;
```

Reply:
yes. at the cost of some readability (imo). but changed.